dapps/marketplace/src/components/Earning.js

```diff
@@ -1,44 +1,79 @@
 import React from 'react'
 import { fbt } from 'fbt-runtime'
 
-const Earnings = ({ total = 0, earned = 0, large }) => (
-  <div className={`earnings-progress${large ? ' large' : ''}`}>
-    <div className="title">
-      <fbt desc="Earnings.Earnings">Earnings</fbt>
-      <div className="total-rewards">{total}</div>
-    </div>
-    <div className="progress">
-      <div
-        className="progress-bar"
-        style={{ width: `${(earned / total) * 100}%` }}
-      />
+const Earnings = ({ total = 0, earned = 0, large, title, showTotal }) => {
+  const titleEl = title ? title : <fbt desc="Earnings.Earnings">Earnings</fbt>
+
+  const earningTitle = !showTotal ? (
+    <span className="earned">{earned}</span>
+  ) : (
+    fbt(
+      fbt.param('earned', <span className="earned">{earned}</span>) +
+        ' of ' +
+        fbt.param('total', total),
+      'Earnings.earnProgress'
+    )
+  )
+  return (
+    <div className={`earnings-progress${large ? ' large' : ''}`}>
+      <div className="title">
+        {titleEl}
+        <div className="total-rewards">{earningTitle}</div>
+      </div>
+      <div className="progress">
+        <div
+          className="progress-bar"
+          style={{ width: `${(earned / total) * 100}%` }}
+        />
+      </div>
     </div>
-  </div>
-)
+  )
+}
 
 export default Earnings
 
 require('react-styl')(`
   .earnings-progress
-    font-size: 18px
```

Comment:
@nick wants to have all font sizes in px until we move to a better solution

Reply:
Didn't know that. I have everything in rem right now. Should I convert them back to px? @nick

Reply:
It's not a big deal for now... but I want to replace all font sizes with vars so it's easier to be consistent throughout. In the meanwhile I've been using px for font sizes and rem for everything else